### PR TITLE
336377: added method/view stubs

### PR DESCRIPTION
**Pull request by AtosSaqlain**

### CODE REVIEW

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Views/Accreditation/CheckYourAnswers.cshtml [hunk 1]
```diff
@@ -18,6 +18,10 @@
 <form method="post">
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds-from-desktop">
+            <p class="govuk-body">
+                <a href="/" class="govuk-back-link">@MasterResources.BackLink</a>
+            </p>
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Views/Accreditation/CheckYourAnswers.cshtml]: There's a back link already provided by the layout, and a singleton injected that provides a url. If that link does not exist, then the back does not get shown. but if it does exist, then back is displayed to the user.

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs [hunk 2]
```diff
@@ -226,5 +226,53 @@ public override void OnActionExecuted(ActionExecutedContext context)
...
+        {
+            return View("dischargeConsentNumber");
+        }
+
+        [HttpGet("ExemptionReference")]
+        public async Task<IActionResult> ExemptionReference(
+            Guid? id)
+        {
+            return View("exemptionReference");
+        }
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs:271]: For stubbed methods you didn't need to provide all the views - just returning null would have been suffcient. All we need to do is prove you redirect from your action to the next one - the next one does not need to work in any shapre or form.

Comment (AtosSaqlain) [reply src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs:271]: I'm pretty sure I was following a pattern I observed in another part of the code. Do you want me to remove the views or leave them?

Comment (AtosSaqlain) [reply src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs:271]: Views have been removed

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Views/Accreditation/CheckYourAnswers.cshtml (hunk 1 above)
Comment (AtosSaqlain) [reply src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Views/Accreditation/CheckYourAnswers.cshtml]: Removed

**Review by NikPeters-Atos — APPROVED** [review]

**Pull request merged**